source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -67,5 +80,27 @@ public static void AddActorContext(this IServiceCollection serviceCollection)
             serviceCollection.AddScoped<IActorContext, ActorContext>();
             serviceCollection.AddScoped<IActorProvider, ActorProvider>();
         }
+
+        public static void AddEventPublishing(this IServiceCollection serviceCollection, ServiceBusClient serviceBusClient)
```

Comment:
Perhaps this would come in handy later; 
```suggestion
        public static void AddInternalEventPublishing(this IServiceCollection serviceCollection, ServiceBusClient serviceBusClient)
```